GoogleSignIn/Sources/GIDGoogleUser.m

```diff
@@ -111,14 +89,41 @@ - (void)updateAuthState:(OIDAuthState *)authState
   _authState = authState;
   _authentication = [[GIDAuthentication alloc] initWithAuthState:authState];
   _profile = profileData;
+  _configuration = [[GIDConfiguration alloc] initWithClientID:[self clientID]
```

Comment:
Is there a reason why we wouldn't just set these variables in the initializer? This method does seem to be used elsewhere, and isn't too bulky.

Reply:
`configuration`, being `authState` derived, probably makes sense to implement with a getter, as we've done for `grantedScopes`.

Reply:
@mdmathias this method is used to update `authState` in the addScopes flow without creating a new `GIDGoogleUser` instance.